yamale/command_line.py

```diff
@@ -112,8 +102,12 @@ def main():
     parser.add_argument('--strict', action='store_true',
                         help='Enable strict mode, unexpected elements in the data will not be accepted.')
     args = parser.parse_args()
-    _router(args.path, args.schema, args.cpu_num, args.parser, args.strict)
-    print('Validation success! 👍')
+    try:
+        _router(args.path, args.schema, args.cpu_num, args.parser, args.strict)
+        print('Validation success! 👍')
+    except ValueError as e:
+        print('Validation failed!\n%s' % e.value.message)
```

Comment:
The `ValueError` type doesn't have a `value` attribute. You can probably just use `str(e)` here.

Reply:
It looks that all exceptions have `value` attribute which contains all exception's members
`str(e)` is shorter bu ugly.

Reply:
I get the following exception when running this code in Python 3.7.6:
```
Validating /Users/chrism/workspace/abourree/Yamale/2.yaml...
Traceback (most recent call last):
  File "/Users/chrism/workspace/abourree/Yamale/yamale/command_line.py", line 106, in main
    _router(args.path, args.schema, args.cpu_num, args.parser, args.strict)
  File "/Users/chrism/workspace/abourree/Yamale/yamale/command_line.py", line 87, in _router
    _validate_single(root, schema_name, parser, strict)
  File "/Users/chrism/workspace/abourree/Yamale/yamale/command_line.py", line 58, in _validate_single
    _validate(s, yaml_path, parser, strict)
  File "/Users/chrism/workspace/abourree/Yamale/yamale/command_line.py", line 26, in _validate
    return yamale.validate(schema, data, strict)
  File "/Users/chrism/workspace/abourree/Yamale/yamale/yamale.py", line 49, in validate
    raise YamaleError(results)
yamale.yamale_error.YamaleError: [<yamale.schema.validationresults.ValidationResult object at 0x10b714050>]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/chrism/workspace/abourree/Yamale/venv/bin/yamale", line 11, in <module>
    load_entry_point('yamale', 'console_scripts', 'yamale')()
  File "/Users/chrism/workspace/abourree/Yamale/yamale/command_line.py", line 109, in main
    print('Validation failed!\n%s' % e.value.message)
AttributeError: 'YamaleError' object has no attribute 'value'
```

Reply:
`str(e)` cuts on first \n

Reply:
When you fix the inheritance of YamaleError, you should get the full message. For example:
```
$ python
Python 3.7.6 (default, Dec 30 2019, 19:38:28) 
[Clang 11.0.0 (clang-1100.0.33.16)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> e = ValueError('what\nnow')
>>> print(str(e))
what
now
```